tests/vsphere_test.go

```diff
@@ -65,7 +83,7 @@ func vsphereTestInit() {
 	}
 
 	sort.Slice(selected, func(i, j int) bool {
-		return strings.Compare(selected[i].Build, selected[j].Build) > 0
+		return getBuildNumberFromString(selected[i].Build) > getBuildNumberFromString(selected[j].Build)
```

Comment:
you have to use the new name :)

Reply:
i should really not do to things at the same time, it starts to get embarrassing :laughing: